SharpBucket/V2/EndPoints/DiffParameters.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SharpBucket.V2.EndPoints
+{
+    /// <summary>
+    /// Parameters for RepositoryResource.GetDiff().
+    /// </summary>
+    public class DiffParameters: ParametersBase
+    {
+        /// <summary>
+        /// Number of context lines for each difference; default is 3.
+        /// </summary>
+        public byte Context { get; set; } = 3;
+        /// <summary>
+        /// The paths of specific files to diff.
+        /// </summary>
+        public IEnumerable<string> Paths { get; set; } = new List<string>();
```

Comment:
Why do we initialise that property with a list ?
Since it will be cast as IEnumerable, we wont be able to edit, and an array will be more adapted than a list
```suggestion
        public IEnumerable<string> Paths { get; set; } = new string[0];
```

But since we allow caller to set that property to null, why not just stick to the null default value ?
```suggestion
        public IEnumerable<string> Paths { get; set; }
```

Reply:
OK, exposing it as a `List<>` could be also a good solution in such class